### PR TITLE
Link concepts which became recently added as concept sets

### LINKED ISSUE #408: Check for reduction of <NA> concepts.
maybe by automatically re-linking all NA concepts in all lists and proposing changes.

This could potentially even become a new command.

### PR DESCRIPTION
**Pull request by xrotwang**
- linked old un-linked concepts to recent concept sets
- added this task to RELEASE instructions

closes #408

Attached is the remaining list of candidate mappings identified by `concepticon notlinked`
[notlinked.txt](https://github.com/clld/concepticon-data/files/2828563/notlinked.txt)

### CODE REVIEW

**Comment by xrotwang** [comment]
I was very conservative in the mapping - only linking rather recent concept sets (as determined by high ID).

File: concepticondata/conceptlists/BeijingDaxue-1964-905.tsv [hunk 1]
```diff
@@ -611,7 +611,7 @@ BeijingDaxue-1964-905-609	609	藏(收存)	305	verb	daily actions	四、日常操
 BeijingDaxue-1964-905-610	610	說	305	verb	communication	五、交際、事務、人事	動詞	speak	shuō	1623	SPEAK	说
 BeijingDaxue-1964-905-611	611	閑淡	306	verb	communication	五、交際、事務、人事	動詞	talk, make smalltalk	xián dàn	2708	TALK	闲澹
 BeijingDaxue-1964-905-612	612	問	306	verb	communication	五、交際、事務、人事	動詞	ask	wèn	1309	ASK (INQUIRE)	问
-BeijingDaxue-1964-905-613	613	理	307	verb	communication	五、交際、事務、人事	動詞	manage	lǐ			理
+BeijingDaxue-1964-905-613	613	理	307	verb	communication	五、交際、事務、人事	動詞	manage	lǐ	3073	MANAGE	理
```
Comment (chrzyki) [inline concepticondata/conceptlists/BeijingDaxue-1964-905.tsv]: Not sure whether the intended meaning her fits ID 3073.

File: concepticondata/conceptlists/Dolch-1936-220.tsv [hunk 2]
```diff
@@ -64,15 +64,15 @@ Dolch-1936-220-62	62	adverbs	no	1269	NO
 Dolch-1936-220-63	63	adverbs	not	1240	NOT
 Dolch-1936-220-64	64	adverbs	now	1376	NOW
 Dolch-1936-220-65	65	adverbs	off		
-Dolch-1936-220-66	66	adverbs	once		
+Dolch-1936-220-66	66	adverbs	once	2914	ONCE (IN THE PAST)
 Dolch-1936-220-67	67	adverbs	only	1580	ONLY
 Dolch-1936-220-68	68	adverbs	out		
-Dolch-1936-220-69	69	adverbs	so		
+Dolch-1936-220-69	69	adverbs	so	2336	OF THIS KIND (SUCH)
```
Comment (chrzyki) [inline concepticondata/conceptlists/Dolch-1936-220.tsv]: The linking seems reasonable, not sure whether the concept definition is fitting.

File: concepticondata/conceptlists/Ogden-1930-100.tsv [hunk 3]
```diff
@@ -89,7 +89,7 @@ Ogden-1930-100-87	87	even
 Ogden-1930-100-88	88	not	1240	NOT
 Ogden-1930-100-89	89	only	1580	ONLY
 Ogden-1930-100-90	90	quite		
-Ogden-1930-100-91	91	so		
+Ogden-1930-100-91	91	so	2336	OF THIS KIND (SUCH)
```
Comment (chrzyki) [inline concepticondata/conceptlists/Ogden-1930-100.tsv]: Likewise, linking reasonable, not sure whether 2336 is defined that well.

File: concepticondata/conceptlists/BeijingDaxue-1964-905.tsv (hunk 1 above)
Comment (LinguList) [reply concepticondata/conceptlists/BeijingDaxue-1964-905.tsv]: Difficult to say, but we can leave this for the time being.

File: concepticondata/conceptlists/Dolch-1936-220.tsv (hunk 2 above)
Comment (LinguList) [reply concepticondata/conceptlists/Dolch-1936-220.tsv]: It's fine.

**Comment by LinguList** [comment]
Was worried with "swallow", but this is linked to the correct animal now.

**Pull request merged**